src/main/java/crazypants/enderio/machines/machine/crafter/TileCrafter.java

```diff
@@ -297,6 +305,7 @@ public boolean canInteractWith(@Nonnull EntityPlayer var1) {
 
         ItemStack matches = ItemStack.EMPTY;
         IRecipe recipe = CraftingManager.findMatchingRecipe(inv, world);
```

Comment:
This lookup also isnt necessary anymore right ?

Reply:
updateCraftingOutput is the method called when the crafting pattern slots change, so this is how the cache is refreshed